composer.json

```diff
@@ -19,14 +29,21 @@
         "dealerdirect/phpcodesniffer-composer-installer": "^0.4",
         "sirbrillig/phpcs-variable-analysis": "^2.0",
         "wimg/php-compatibility": "^8.0",
-        "wp-coding-standards/wpcs": "^0.14.0"
+        "wp-coding-standards/wpcs": "^0.14.0",
+        "phpunit/phpunit": "^6.2",
+        "antecedent/patchwork": "2.0.8",
```

Comment:
This seems to be a rather specific version. I think `2.0.9` is out, but this won't allow for that. Is that intentional?
Any reason not to let the patchwork dependency in Brain Monkey handle this?

Reply:
@GaryJones Good points. Removed to let Brain Monkey handle its own dependencies.  Pushing shortly.